microapidev/comment-microapi
# [TEST]RepliesEndpointTests GET /comments/{commentId}/replies/{replyId}:Create test suite to test the endpoint and it's expected behaviour

**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**
Fix  #180
Fixes #issue_number (replace issue_number with the issue number this PR solves)

Summarise the main tasks handled in the pull request
Create test suite to for the /comments/{commentId}/replies/{replyId} endpoint
**description of Task to be completed?**
Create test suite to test the /comments/{commentId}/replies/{replyId} endpoint and it's expected behaviour
Describe (in detail) what the task completed in the pull request does as per the relevant task
I wrote a test for this edpoint
**How should this be manually tested?**

Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)
Just run `npm run test`, if the endpoint works as expected,it should return a reply when the get reply by replyId test runs.
**Any background context you want to provide?**

Any pertinent information that should be considered

**What is the link to the issue on Github?**
https://github.com/microapidev/comment-microapi/issues/180
[Issue #14](https://github.com/microapidev/comment-microapi/issues/14)

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here
